protocol-designer/src/components/DeckSetup/index.tsx

```diff
@@ -533,7 +533,11 @@ export const DeckSetup = (): JSX.Element => {
   ).filter(aE => WASTE_CHUTE_CUTOUT.includes(aE.location as CutoutId))
```

Comment:
does this not also need to heck that the equipment name is a wastechute?
```suggestion
  ).filter(aE => WASTE_CHUTE_CUTOUT.includes(aE.location as CutoutId) && aE.name === 'wasteChute'))
```